common_spree_dependencies.rb

```diff
@@ -6,7 +6,7 @@
 gem 'json'
 gem 'multi_json'
 gem 'mysql2'
-gem 'pg'
+gem 'pg', '~> 0.11'
```

Comment:
Why is this change needed? Since in OFN we use 0.21.0 I would stick to that one to be sure things work on our version.

Reply:
It's the one used here (see the Gemfile.lock), if I don't specify `'~> 0.11'` it uses 1.1.x

Reply:
'< 1.1.0' raised the same error, '<1.0' seem to work fine (it uses 0.21)